test/jasmine/tests/click_test.js

```diff
@@ -817,6 +928,588 @@ describe('Test click interactions:', function() {
     });
 });
 
+
+describe('Test click interactions on a pie plot:', function() {
```

Comment:
Also, can you move these cases to https://github.com/plotly/plotly.js/blob/master/test/jasmine/tests/hover_pie_test.js

I like separating our test cases per subplot types.